lib/Widgets/TransmissionListEntry.vala

```diff
@@ -57,6 +57,7 @@ public class Drop.Widgets.TransmissionListEntry : Gtk.Grid {
 
         secondary_label = new Gtk.Label ("");
         secondary_label.halign = Gtk.Align.START;
+        secondary_label.hexpand = true;
```

Comment:
Moved hexpand to the secondary label to:
- Avoid buttons to expanding more than needed (Causing un-uniformity) 
- didn't add it to primary lable to not break the elipsize